src/Microsoft.ApiDesignGuidelines.Analyzers/UnitTests/CollectionsShouldImplementGenericInterfaceTests.cs

```diff
@@ -12,12 +12,366 @@ public class CollectionsShouldImplementGenericInterfaceTests : DiagnosticAnalyze
     {
         protected override DiagnosticAnalyzer GetBasicDiagnosticAnalyzer()
         {
-            return new BasicCollectionsShouldImplementGenericInterfaceAnalyzer();
+            return new CollectionsShouldImplementGenericInterfaceAnalyzer();
         }
 
         protected override DiagnosticAnalyzer GetCSharpDiagnosticAnalyzer()
         {
-            return new CSharpCollectionsShouldImplementGenericInterfaceAnalyzer();
+            return new CollectionsShouldImplementGenericInterfaceAnalyzer();
+        }
+
+        [Fact]
+        public void Test_WithCollectionBase()
```

Comment:
The convention I see most often for analyzer test method names is something like

```
CA1010_CSharp_DiagnosticWithCollectionBase
...
CA1010_CSharp_NoDiagnosticWithGenericCollection
```
This sorts the method names nicely in the Test Explorer window.
```